contracts/vaults/Params.sol

```diff
@@ -6,9 +6,9 @@ import "./Base.sol";
 contract Params is Base {
 
     /**
-    * @notice Set new committee address. Can be called by existing committee if it had checked in, or
-    * by the governance otherwise.
-    * @param _committee new committee address
+    * @notice Set new committee address. Can be called by existing committee
+    * if it had previously checked in, or by the governance otherwise.
```

Comment:
Not correct, committee can call it even if it hasn't checked in. This limit is only for the owner if the committee did check in